packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -1582,6 +1582,7 @@ void main() {
     Widget runTest() {
       int _currentIndex = 0;
       return MaterialApp(
+        theme: ThemeData(splashFactory: InkSplash.splashFactory),
```

Comment:
I suppose you are doing this temporarily for tests to pass?
In proposal you mention:

> Default SplashFactory: As a quick follow-up to the first PR, we should also change the default splash factory to the InkRipple.splashFactory, which is going to match the material animation for all Android APIs going forward. Currently, InkSplash.splashFactory is the default, which is an outdated choice, that's getting more outdated by the day.

After this change, all similar replacements of `InkSplash.splashFactory` in tests should be removed

Reply:
Ah yes good point—I remember doing this, because at the time, I didn't understand how to update/add goldens (which is how we're testing this widget at the moment).

I'll go through and add another set of tests for the bottom navigation bar, that uses a new set of goldens, so that eventually, if we wanted to delete any tests related to the InkSplash.splashFactory, that could be a quick deletion 👍🏻